<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizCreateAccountDialog</class>
 <widget class="QDialog" name="WizCreateAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Account</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>User ID (Email):</string>
   </property>
   <property name="buddy">
    <cstring>editUserId</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUserId">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="buddy">
    <cstring>editPassword</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPassword">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPassword2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
   <property name="buddy">
    <cstring>editPassword2</cstring>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>140</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editUserId</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editPassword2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WizCreateAccountDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WizCreateAccountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
